samples/FBSDKPlacesSample/FBSDKPlacesSample/PlaceListViewController.m

```diff
@@ -197,9 +229,6 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
   else if ([place.confidence isEqualToString:@"high"]) {
     cell.textLabel.textColor = [UIColor greenColor];
   }
-  else {
```

Comment:
This is for the current place confidence, and without it, you may get re-used cells that do not black when they should be.